File: ProjectManagerApi/TaskManagerApi/Models/Project.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace TaskManagerApi.Models
{
    public class Project
    {
        public int ProjectId { get; set; }
        public string Name { get; set; }
        public string StartDate { get; set; }
        public string EndDate { get; set; }
        public int Priority { get; set; }
    }
}

File: ProjectManagerAngular/src/app/project.ts
export class Project {
    ProjectId: number = 0;
    Name: string = "";
    StartDate: string = new Date().toISOString().split('T')[0];
    EndDate: string = new Date().toISOString().split('T')[0];
    Priority: number = 0;
    ManagerId!: number;
  }